utils/text_formatting.tsx

```diff
@@ -262,6 +271,16 @@ export function formatText(
         output = `<span class="all-emoji">${output.trim()}</span>`;
     }
 
+    if (options.postId) {
+        // unwrap the output from the closing p tag and add a span that will serve as a
+        // palceholder for `messageToHtmlComponent` function later on
+        if (output.endsWith('</p>')) {
+            output = `${output.slice(0, -4)} <span data-edited-post-id='${options.postId}'></span></p>`;
```

Comment:
Hello @michelengelen, sorry to bother you but I wonder if the extra spaces here were intended or can be changed to:
```diff
- output = `${output.slice(0, -4)} <span
+ output = `${output.slice(0, -4)} <span

- output += ` <span
+ output += `<span
```

Though no perceived side-effect in actual UI as observed by the user, our UI automation tests failed a lot with this new format due to the extra space rendered at the last position of the message.

Please take a look and let me know what you think. Thanks!

Reply:
I think this is making also some Apps Framework e2e tests to fail.

Reply:
it is ... i am currently fixing this in #8964